pkgs/servers/tracing/honeycomb/refinery/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "honeycomb-refinery";
+  version = "1.18.0";
+
+  src = fetchFromGitHub {
+    owner = "honeycombio";
+    repo = "refinery";
+    rev = "v${version}";
+    sha256 = "sha256-aWam16muUFA7Z0gTArYuNsxanlOaRlYJma7LN0u1XlE=";
+  };
+
+  NO_REDIS_TEST = true;
+
+  patches = [
+    # Allows turning off the one test requiring a Redis service during build.
+    # We could in principle implement that, but it's significant work to little
+    # payoff.
+    ./0001-add-NO_REDIS_TEST-env-var-that-disables-Redis-requir.patch
+  ];
+
```

Comment:
It would be simpler to just delete the test file.
```suggestion
```

Reply:
That's fair, but the test file also contains other tests. I don't think this patch is likely to ever break anyway.

Reply:
Looks like it has another test case inside the same text fixture, which is weird. I guess we can keep the patch then.